engine/baml-lib/baml/tests/validation_files/strings/unquoted_strings.baml

```diff
@@ -8,28 +8,46 @@ client<llm> Hello {
   }
 }
 
-// error: Error validating: This line is not a valid field or attribute definition. A valid property may look like: 'myProperty "some value"' for example, with no colons.
+// error: Error validating Client "Hello": This field declaration is invalid. It is either missing a name or a type.
+//   -->  strings/unquoted_strings.baml:3
+//    | 
+//  2 |   provider baml-openai-chat
+//  3 |   options {
+//    | 
+// error: Error validating: This line is not a valid field or attribute definition.
 //   -->  strings/unquoted_strings.baml:3
 //    | 
 //  2 |   provider baml-openai-chat
 //  3 |   options {
 //  4 |     thing hello'world
 //    | 
-// error: Error validating: This line is not a valid field or attribute definition. A valid property may look like: 'myProperty "some value"' for example, with no colons.
```

Comment:
feels like the previous error was better because it told you how to fix it. Usuallly people add colons.